mllib/src/main/scala/org/apache/spark/ml/feature/Instance.scala

```diff
@@ -26,7 +26,7 @@ import org.apache.spark.ml.linalg.Vector
  * @param weight The weight of this instance.
  * @param features The vector of features for this data point.
  */
-private[ml] case class Instance(label: Double, weight: Double, features: Vector)
+private[spark] case class Instance(label: Double, weight: Double, features: Vector)
```

Comment:
Where does this get used outside ml? I'd imagine its use can and should stay within the ml package

Reply:
it was used in mllib/src/main/scala/org/apache/spark/mllib/tree/RandomForest.scala when passing labeledpoint to new-style ml random forest, which required converting to ml Instance from mllib LabeledPoint.  Hence, I've added another run method on RandomForest in ml that takes the old-style mllib LabeledPoints and converts to Instances.